db/orders.js

```diff
@@ -157,18 +160,18 @@ async function attachProductsToOrder(orders) {
 async function addProductToCart({ productId, orderId, price, quantity }) {
```

Comment:
do we need curly braces? i feel that it may cause an error - since you are later use not all of them in some functions

Reply:
yes we do because when we fetch them as an object. I don't use curly braces when functions take only one parameter, which will not cause any issue.